packages/geo/src/lib/feature/shared/feature.utils.ts

```diff
@@ -71,6 +72,10 @@ export function featureToOl(
 
   olFeature.set('_entityRevision', getEntityRevision(feature), true);
 
+  if (feature.icon) {
+    olFeature.icon = feature.icon;
```

Comment:
Same comment as above

Reply:
Because we need to "track" the feature icon to display it on the map when there is single point element. The icon is added to the datasource of the resulting layer.

Reply:
See above

Reply:
done